git-tutorial.markdown

```diff
@@ -98,11 +98,77 @@ If you have your old installation under `~/alice` then we suggest you use a
 different directory to avoid confusion.
 
 
+### I don't use aliBuild
+
+If you do not use aliBuild you can keep using your previous installation
+commands without any modification. You need however to clone the ALICE
+repositories from GitHub:
+
+```bash
+cd ~/alice
+git clone https://github.com/alisw/AliRoot
+git clone https://github.com/alisw/AliPhysics
+```
+
+
+### I have participated to the GitHub test
+
+> These instructions are only valid from the time the GitHub migration has been
+> completed!
+
+If you have participated to the GitHub test and you want to sync with the
+current official AliRoot/AliPhysics and alidist repositories from upstream,
+starting from the day when Git migration is effective you can follow this
+procedure and you will save some build time.
+
+Assuming you have your GitHub test installation under `~/alice`, update your
+recipes:
+
+```bash
+cd ~/alice
+rm -rf alidist/
+aliBuild init
+```
+
+The `aliBuild init` command will re-download the production version of the
+build recipes. Now move into your AliRoot and AliPhysics directories and update
+them. Note that the following operation is **destructive** and you will lose
+any local change for good!
+
+```bash
+cd ~/alice/AliPhysics
+git fetch --all
+git checkout master
+git reset --hard origin/master
+```
+
+Do the same for AliRoot. If you also develop AliRoot and AliPhysics, you will
+need to open pull requests. As a one-time operation, forcibly sync your GitHub
+fork of AliPhysics with the upstream one:
+
+```bash
+git push -f <your-github-username>
+```
+
+The `-f` option is not normally used in production. Once you have done that, you
```

Comment:
The rest of the documentation (the basic one!) discourages the use of `-f`. Experience tells us that people do disastrous things with `-f`. As discussed many times in the working group, we don't want to give the idea that "if you always use `-f` the command will never fail, so I am gonna use it all the time."

By separating this from the rest of the documentation we are clearly separating the part where people can safely do `-f` from the part where they can't.